bhs_api/item.py

```diff
@@ -189,6 +189,15 @@ def enrich_item(item, db=None, collection_name=None):
     if pictures:
         main_image_id = None
         for image in pictures:
+            # Add 'PictureUrl' to all images of an item
+            image_id = image['PictureId']
+            if item.has_key('bagnowka'):
```

Comment:
better to also check if the value is True, not only that it exists
can be done like this:
```
if item.get('bagnowka'):
```